template/azurePipelines/kustomize/draft.yaml

```diff
@@ -0,0 +1,30 @@
+variables:
+  - name: "PIPELINE_NAME"

```

Comment:
to keep it consistent, let's not use the separators unless you have a specific reason. Can just keep it "PIPELINENAME"
https://github.com/Azure/draft/blob/main/template/workflows/helm/draft.yaml

Reply:
removed the separators to maintain consistency with the other templates